client/src/components/TimePicker.jsx

```diff
@@ -0,0 +1,60 @@
+import React, {PropTypes} from "react";
+import { makeStyles } from "@material-ui/core";
+import { Button, Box, Typography, Tab, Tabs } from "@material-ui/core"
+
+
+const TimePicker = (props) => {
+  const classes = useStyles();
+  const [value, setValue] = React.useState(10);
+
+  let button = <Tab label={<Button className={classes.button} variant="outlined" size="large">Time</Button>} />
+  let items = []
+  let i;
+  for (i = 0; i < 10; i++) {
+    items.push(button);
+  }
+  const handleChange = (event, newValue) => {
+    setValue(newValue);
+  };
+
+  return (
+    <React.Fragment>
+      <div className={classes.root}>
+        <Tabs
+          orientation="vertical"
+          variant="scrollable"
+          value={value}
+          onChange={handleChange}
+          aria-label="Vertical tabs example"
+          className={classes.tabs}
+        >
+          { items }
+        </Tabs>
+      </div>
+    </React.Fragment>
```

Comment:
if possible I would remove this `React.Fragment` as it's already being wrapped arouns with the `dev` element